src/include/rados/librados.h

```diff
@@ -3124,7 +3124,28 @@ CEPH_RADOS_API void rados_read_op_omap_get_vals(rados_read_op_t read_op,
 				                const char *filter_prefix,
 				                uint64_t max_return,
 				                rados_omap_iter_t *iter,
-				                int *prval);
+				                int *prval)
+  __attribute__((deprecated)); /* use v2 below */
```

Comment:
seems like the originals are useful convenience functions that shouldn't be deprecated now that they're looping on the client side